compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -152,7 +152,7 @@ FailureOr<ParameterSetting> ParameterSetting::create(linalg::LinalgOp linalgOp,
   const uint64_t K = lhsShape[1];
 
   FailureOr<unsigned> maybeScaleFactor =
-      getTilingScaleFactor(initType.getElementType());
+      getTilingScaleFactor(lhsType.getElementType());
```

Comment:
So we are to not base it on the accumulator but the input element types ? I believe this is in-line with the observations made for `INF NAN` issue ?

Reply:
That's correct, I believe this is the change which caused `INF NAN`. This is still a very experimental PR!